session/object.go

```diff
@@ -166,6 +166,11 @@ func (x *Object) Sign(signer neofscrypto.Signer) error {
 	return x.sign(signer, x.writeContext)
 }
 
+// SignedData returnes actual payload which would be signed, if you call [Object.Sign] method.
+func (x *Object) SignedData() []byte {
```

Comment:
i can offer a pretty indicative scenario for unit test:
1. fill token
2. call `SignedData`
3. calculate signature manually
4. call `Sign` with static signer
5. assert `Verify` passes